src/texture/any.rs

```diff
@@ -1426,7 +1426,7 @@ impl<'a> TextureAnyImage<'a> {
         let mut ctxt = self.texture.context.make_current();
 
         let mut data = Vec::new();
-        ops::read(&mut ctxt, &fbo::RegularAttachment::Texture(*self), &rect, &mut data, false);
+        let _ = ops::read(&mut ctxt, &fbo::RegularAttachment::Texture(*self), &rect, &mut data, false);
```

Comment:
I remember leaving these warnings on purpose so that they would get fixed in a proper way.